actin/operational/load_actin_operational_data

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+source message_functions || exit 1
+source database_functions || exit 1
+source locate_files || exit 1
+
+patient=$1 && shift
+
+if [[ -z "${patient}" ]]; then
+    error "This script takes a patient as an argument"
+fi
+
+actin_database_sql="$(locate_actin_database_sql_script)"
+actin_database_views_sql="$(locate_actin_database_views_sql_script)"
+
+db_host="actin-sql.prod"
+db_port=3306
+db_user="writer"
+db_pass=$(gcloud secrets versions access latest --secret="prod-sql-writer-password")
+db_name=actin_operations
+
+database_exists=$(mysql -u "$db_user" -p"$db_pass" -h "$db_host" -e "SHOW DATABASES LIKE '${db_name}';" | grep "$db_name")
+
+if [ -z "$database_exists" ]; then
+  echo "Database '$db_name' does not exist. Loading SQL script..."
+  mysql -u "$db_user" -p"$db_pass" -h "$db_host" -e "CREATE DATABASE $db_name;"
+  mysql -u "$db_user" -p"$db_pass" -h "$db_host" "$db_name" < $actin_database_sql
+  mysql -u "$db_user" -p"$db_pass" -h "$db_host" "$db_name" < $actin_database_views_sql
+
+else
+  echo "Database '$DATABASE_NAME' already exists."
+fi
+
+treatment_match_json="$(locate_actin_treatment_match_json ${patient})"
+
+java -cp /data/actin/system/actin.jar com.hartwig.actin.database.algo.TreatmentMatchLoaderApplication \
```

Comment:
You should find the jar via ```actin_jar="$(locate_pilot_actin)"```

(that being said, maybe time to rename that function :)